hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxSigs.java

```diff
@@ -129,7 +129,16 @@ static byte[] calculateSignableMessage(EthTxData ethTx) {
                 ethTx.callData(),
                 new Object[0]
             });
-            case EIP2930 -> throw new IllegalArgumentException("Unsupported transaction type " + ethTx.type());
+            case EIP2930 -> RLPEncoder.encodeSequentially(Integers.toBytes(1), new Object[] {
```

Comment:
Didn't we just see all these unsupported lists of fields being encoded over at `EthTxData`?  And here they are again.  Hope they're in the same order!

My suggestion: File an issue to refactor these two files, if not rewrite them entirely.

Reply:
I didn't want to risk bug introduction via a refactor, I also felt we couldn't better organize items to increase sharing and readability.
I believe we should get an improved layout in modularization so no need to revamp this in mono service.

Reply:
Instead of creating new methods, we could use `ethTx.encodeTx()` right? Or am I missing something?